includes/Embed_Base.php

```diff
@@ -55,6 +62,23 @@ public function init() {
 		wp_register_script( self::STORY_PLAYER_HANDLE, 'https://cdn.ampproject.org/amp-story-player-v0.js', [], 'v0', false );
 		wp_register_style( self::STORY_PLAYER_HANDLE, 'https://cdn.ampproject.org/amp-story-player-v0.css', [], 'v0' );
 
+		// Registering a style without a `src` allows us to just use the inline style below
+		// without needing an external stylesheet.
+		wp_register_style(
+			self::SCRIPT_HANDLE,
+			'',
+			[],
+			WEBSTORIES_VERSION
+		);
+
+		if ( file_exists( WEBSTORIES_PLUGIN_DIR_PATH . 'includes/assets/embed.css' ) ) {
```

Comment:
```suggestion
		if ( is_readable( WEBSTORIES_PLUGIN_DIR_PATH . 'includes/assets/embed.css' ) ) {
```
Massive nit pic, but it is best to use `is_readable`, as just because the file exists, doesn't mean php can read it. :P